substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -1779,6 +1785,15 @@ pub mod pallet {
 			MinCommission::<T>::put(new);
 			Ok(())
 		}
+
+		/// Sets the fraction of the maximum staked rewards of the era's inflation.
+		#[pallet::call_index(26)]
+		#[pallet::weight(T::WeightInfo::set_max_staked_rewards())]
+		pub fn set_max_staked_rewards(origin: OriginFor<T>, new: Percent) -> DispatchResult {
```

Comment:
Could also be part of `set_staking_configs`.

Reply:
conclusion on this?

Reply:
Removed this callable and now the max staked rewards can be set via the `set_staking_configs`. To keep track of removing this (and others) configs once we have the params pallet: https://github.com/paritytech/polkadot-sdk/issues/2874